packages/keyring-api/src/api/account.ts

```diff
@@ -20,13 +20,21 @@ export enum BtcAccountType {
   P2wpkh = 'bip122:p2wpkh',
 }
 
+/**
+ * Supported Solana account types.
+ */
+export enum SolAccountType {
+  Eoa = 'solana:system',
```

Comment:
Is this `system` a known thing in Solana? Since we are naming the value `Eoa`, I was wondering if this should rather be:

```ts
Eoa = 'solana:eoa',
```

Or:

```ts
System = 'solana:system',
```

Reply:
Hehehe I had it like that `solana:eoa` at the begining 😆 but changed to be `solana:system` since it's created by the System Program and referred as a "System Account" in some [docs](https://solana.com/docs/core/accounts) of solana. But for this case I also think that we can go with `Eoa = 'solana:eoa', It's clearer. 👍🏼 
`